### PR TITLE
Fixes #4681 - [Annotations][Texttool] Add placeholder text

### LINKED ISSUE #4681: [Annotations] Add placeholder text
When a user selects text, we should add the word "Hello" as a default input. It's confusing to not see any placeholder text and is sometimes hard to grok where the cursor is.

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

File: locales/en-US/server.ftl [hunk 1]
```diff
@@ -231,6 +231,9 @@ annotationTextConfirmButton =
     .title = Confirm
 annotationTextCancelButton =
     .title = Cancel
+# Texttool input textbox placeholder
```
Comment (flodolo) [inline locales/en-US/server.ftl]: I believe this comment needs rewording, right now it feels like a dev note. Can you point me to a mock-up or screenshot?

**Comment by punamdahiya** [comment]
@6a68 please review #4681 fix which is blocker identified by John. Thanks!

**Comment by punamdahiya** [comment]
@flodolo  Attaching screenshot of text input with placeholder. This is seen when user clicks Texttool icon and enters  add text screen
screenshot-texttool-placeholder

**Comment by flodolo** [comment]
Then something like "Default placeholder used in input field when adding annotations" should work.

**Comment by punamdahiya** [comment]
Marked hold because of typo in variable name hasInput, updating now

**Comment by punamdahiya** [comment]
Updated and removed hold. Ready for review.

**Pull request merged**